src/main/java/com/uwetrottmann/tmdb2/DiscoverMovieBuilder.java

```diff
@@ -1,3 +1,20 @@
+/*
```

Comment:
Again, please read [the contributing guidelines](https://github.com/UweTrottmann/tmdb-java/blob/master/CONTRIBUTING.md) and [the actual license](https://github.com/UweTrottmann/tmdb-java/blob/master/UNLICENSE) and make sure you agree to those terms.

There are no "credits" or attributions inside the source code. Only the Git log and the [GitHub contributors view](https://github.com/UweTrottmann/tmdb-java/graphs/contributors) will mention your contribution.

If you add new files they should have no license statement on top, also no author reference or similar.

If you change an existing file completely, you can also remove an old license statement at the top.